src/Shared/Components/CICDHistory/Artifacts.tsx

```diff
@@ -259,26 +250,28 @@ const Artifacts = ({
                     </div>
                 </CIListItem>
             )}
-            {blobStorageEnabled && getArtifactPromise && (type === HistoryComponentType.CD || isArtifactUploaded) && (
-                <CIListItem
-                    type="report"
-                    hideImageTaggingHardDelete={hideImageTaggingHardDelete}
-                    isSuperAdmin={isSuperAdmin}
-                    renderCIListHeader={renderCIListHeader}
-                >
-                    <div className="flex column left">
-                        <div className="cn-9 fs-14">Reports.zip</div>
-                        <button
-                            type="button"
-                            onClick={handleArtifact}
-                            className="anchor p-0 cb-5 fs-12 flex left pointer"
-                        >
-                            Download
-                            <Download className="ml-5 icon-dim-16" />
-                        </button>
-                    </div>
-                </CIListItem>
-            )}
+            {blobStorageEnabled &&
+                getArtifactPromise &&
+                (type === HistoryComponentType.CD || isArtifactUploaded || isJobView) && (
```

Comment:
When CIDetails is called from inside JobDetails isJobView is set to true. So isJobView is an indicator of whether we are in JobDetails.